src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs

```diff
@@ -575,6 +560,33 @@ private void CorrectImagePath(WorkItem wi, WiItem wiItem, WiRevision rev, ref st
             }
         }
 
+        private void CorrectUserAndDate(WiRevision rev, WorkItem wi)
```

Comment:
Not very fond of the naming... but I couldnt figure out anything better. And I would split it up.

CorrectClosedByAndClosedDate(......)
CorrectActivatedByAndActivatedDate(...)

Then you dont have to deepdive into the functions do know what they are doing.

Reply:
Yep! I didn't like it either, but felt the method was very complex, so better to split it.

I addressed your comment here: https://github.com/solidify/jira-azuredevops-migrator/pull/647/commits/f34263fa6c0c63bd1ccf6f055e26e6c9b3fa4c7a